### DISCUSSION TITLE
Script no longer moves on to second page

### DISCUSSION THREAD

**Discussion by jskoiz** [victornpb/undiscord#284] [1. Questions & Answers]
Seems like discord made some changes where the first page of results are all thats deleted. Confirmed in FF and Chrome same behavior 

<img width="697" alt="Screen Shot 2021-12-11 at 9 59 21 AM" src="https://user-images.githubusercontent.com/20649937/145689968-f69a075b-11ec-4023-bcce-dd35a6657a96.png">

**Comment by eIiminated** [comment]
can confirm that it's broken on FF 95.0

**Comment by andrewkm** [comment]
Same, broken here on Chrome as well.
Alternative is to continue punching in a new "Before message with ID" continuously.
Would be nice if you could automated that @victornpb

**Comment by andrewkm** [comment]
Seems to have fixed itself on my end...
Working just as it was before.
Thank god it's not some Discord limitation they had, looks like it was just some bug.

**Reply by Wanted135** [reply]
It took 4 days to fix itself ?

**Reply by andrewkm** [reply]
Who knows man, but it works fine now.
It was broken for a while.

**Reply by Wanted135** [reply]
> Who knows man, but it works fine now. It was broken for a while.

In the same DM or another one ?

**Reply by andrewkm** [reply]
It works fine everywhere.

**Reply by Wanted135** [reply]
This issue occurs only when deleting msgs manually directly in the search bar

**Reply by Wanted135** [reply]
On a specific DM/Channel

**Reply by andrewkm** [reply]
Works fine to me.

**Reply by eIiminated** [reply]
I'd have to test it out for myself tomorrow

**Reply by Wanted135** [reply]
> I'd have to test it out for myself tomorrow

Try deleting msgs directly from the search bar and you will see it won't work, basically script will detect that some pages have 0 msgs left to delete, meaning the script just stop instead of going to the next page

**Reply by eIiminated** [reply]
> > I'd have to test it out for myself tomorrow
> 
> Try deleting msgs directly from the search bar and you will see it won't work, basically script will detect that some pages have 0 msgs left to delete, meaning the script just stop instead of going to the next page

search bar? could you provide pictures?

**Reply by Wanted135** [reply]
> > > I'd have to test it out for myself tomorrow
> > 
> > 
> > Try deleting msgs directly from the search bar and you will see it won't work, basically script will detect that some pages have 0 msgs left to delete, meaning the script just stop instead of going to the next page
> 
> search bar? could you provide pictures?

https://i.imgur.com/PNdLH7T.png

**Comment by Wanted135** [comment]
Ok I fixed it, in line 277  add this :
BEFORE : 
if (messagesToDelete.length > 0) {
AFTER
if (messagesToDelete.length >= 0) {

**Reply by Wanted135** [reply]
You will need to stop it manually when there's nothing more to delete I think this is really a scuffed fix as I know absolutely nothing about coding

**Reply by andrewkm** [reply]
I still don't understand your problem as I cannot reproduce it and everything works fine here.

**Reply by Wanted135** [reply]
ok?
